Dockerfile

```diff
@@ -55,7 +56,9 @@ RUN curl -L https://github.com/krallin/tini/releases/download/v0.6.0/tini > tini
 RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
     python2 get-pip.py && \
     python3 get-pip.py && \
-    rm get-pip.py
+    rm get-pip.py && \
+    pip2 --no-cache-dir install requests && pip2 --no-cache-dir install requests[security] && \
```

Comment:
You're right. I was having some issue where I was only seeing requests in `python3` not `python2`. Regardless, this had nothing to do with it. I will remove it.